quickstart.Dockerfile

```
@@ -17,7 +17,7 @@ USER root
 
 # Install packages
 RUN apt update
-RUN apt -y install curl python3.9 python3.9-dev python3.9-distutils gcc gnupg apache2-utils sudo openssl systemctl
+RUN apt -y install curl python3.9 python3.9-dev pyton3.9-distutils gcc gnupg apache2-utils sudo openssl systemctl
```

```suggestion
RUN apt -y install curl python3.9 python3.9-dev python3.9-distutils gcc gnupg apache2-utils sudo openssl systemctl
```
I'm assuming this was an oversight